component/src/main/java/org/wso2/extension/siddhi/store/elasticsearch/ElasticsearchEventTable.java

```diff
@@ -901,8 +910,21 @@ private void createIndex() {
             throw new ElasticsearchEventTableException("Error while creating indices for table id : '" +
                     tableDefinition.getId(), e);
         } catch (ElasticsearchStatusException e) {
-            logger.debug("Elasticsearch status exception occurs while creating index for table id: " +
+            logger.error("Elasticsearch status exception occurs while creating index for table id: " +
```

Comment:
Shall we fix the "occurs" typo as well? (Elasticsearch status exception occurred while...)

Reply:
Fixed in  0522934